examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,65 @@
+"""
+=========================
+Zoomed-in cutout of a Map
+=========================
+
+This example demonstrates how to showcase a particular region of a full disk
+image within the same figure.
+
+This is a useful for figures that want to provide an overview of your subject.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.patches import ConnectionPatch
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# We will start off by defining a region of interest that we want to focus on.
+# In this case, we want to focus on the center of the Sun.
+length = 200 * u.arcsec
+x0 = -50 * u.arcsec
+y0 = -30 * u.arcsec
+
+###############################################################################
+# Now we will create a Map and then crop the region of interest.
+smap = sunpy.map.Map(AIA_171_IMAGE)
+bottom_left = SkyCoord(x0 - length, y0-length,
+                       frame=smap.coordinate_frame)
```

Comment:
```suggestion
bottom_left = SkyCoord(x0, y0, frame=smap.coordinate_frame)
```

To be consistent with above changes to FOV